test/lib/mayaUsd/fileio/testImportChaser.py

```diff
@@ -67,6 +67,24 @@ def PostImport(self, returnPredicate, stage, dagPaths, sdfPaths, jobArgs):
 
             self.editsRecord.append([fnDagNode,strAttr])
 
+        # Get the Sdf path to Dag path matching of all imported prims
+        # Call GetSdfToDagMap() to get the sdfToDagMap, 
+        # which contains SDF path to Dag path mapping
+        print(" ----------------- SdfToDagMap ------------------")
+        allSdfPaths = []
+        allDagPaths = []
+        sdfToDagMap = self.GetSdfToDagMap()
+        for sdfToDag in sdfToDagMap:
+            allSdfPaths.append(sdfToDag.key())
+            allDagPaths.append(sdfToDag.data())
+
+        assert(len(allSdfPaths) == len(allDagPaths))
+
+        for i in range(len(allSdfPaths)):
+            print(("----------primitive ") + str(i))
+            print(str(allSdfPaths[i]))
+            print(str(allDagPaths[i]))
```

Comment:
Why is the test printing stuff - is that just for you to debug?

Reply:
No actually I want to do a demo here to show the customers how to use the feature..

Reply:
If it's not needed I can remove it